AES.SecuritySvc.Tests/SecurityUnitTest.cs

```diff
@@ -14,12 +14,12 @@ public class SecurityUnitTest
         private const string LASTNAME = "TestLast";
         private const string SSN = "11-22-3344";
         private readonly string SSN_CRYPT;
-        private readonly DateTime DOB;
+        private readonly DateTime TEST_DOB;
 
         public SecurityUnitTest()
         {
             SSN_CRYPT = Encryption.Encrypt(SSN);
-            DOB = new DateTime(1970, 1, 1);
+            TEST_DOB = new DateTime(1964, 6, 2);
```

Comment:
Does this work? Usually 1970 is the earliest a date can go. Why the change with the name as well?